src/packets.toit

```diff
@@ -231,6 +177,14 @@ class ConnectPacket extends Packet:
     if password: Packet.encode-string buffer password
     return buffer.bytes
 
+  stringify -> string:
+    return ("Connect: $client-id"
```

Comment:
Drop the `(` and use 4-space indent for the next lines?

Reply:
done.